crates/project_panel/src/project_panel.rs

```diff
@@ -1677,6 +1677,23 @@ impl ProjectPanel {
         let width = self.size(cx);
         let filename_text_color =
             entry_git_aware_label_color(details.git_status, details.is_ignored, is_selected);
+        let project = self.project.read(cx);
+        let tooltip_text = project
```

Comment:
<img width="648" alt="image" src="https://github.com/zed-industries/zed/assets/2690773/4cfe5cc4-3b32-4c3b-82cd-a40723f4c61f">

I'm not very sure this will work nice now: we do not have a good keybindings set-up to work with the project panel, so multiple file selections and other things are commonly done with the mouse/trackpad — having extra pop-ups flashing around seems bad.

I've checked RustRover (Intellij-based JetBrains IDE), Sublime Text, VSCode and Fleet — only the last one has this functionality enabled by default.
So it's not very widespread either, at least, not enabled by default.

Reply:
Also, as it can be seen on my screenshot, added/modified statuses could be shown in the project panel already, and the full path of any entry could be copied with a keybinding — I'm not sure that this tooltip brings much value too.